pixleesdk/src/main/java/com/pixlee/pixleesdk/client/PXLBaseAlbum.java

```diff
@@ -405,6 +424,23 @@ public void onFailure(Call<MediaResult> call, Throwable t) {
             body.put("username", username);
             body.put("approved", approved);
 
+            if(productSKUs!=null && productSKUs.size() > 0){
+                JSONArray arr = new JSONArray();
+                for (String item : productSKUs)
+                    arr.put(item);
+                body.put("product_skus", arr);
+            }
+
+            if(categoryNames!=null && categoryNames.size() > 0){
+                JSONArray arr = new JSONArray();
+                for (String item : categoryNames)
+                    arr.put(item);
+                body.put("category_names", arr);
+            }
+
+            if(connectedUser!=null && connectedUser.length() > 0) {
```

Comment:
we also use length for {} objects?

Reply:
seems like the API for length in this case returns the count of key value pairs in the object

Reply:
that's right! when connectedUser is empty=={}, connectedUser.length() is 0. If you put 4 sets of key-values, then .length() will be 4. The example's .length() is 4.
```
JSONObject connectedUser = new JSONObject();
connectedUser.put("name", "Sample data");
connectedUser.put("age", 73);
connectedUser.put("email", "b@b.com");
JSONArray arr = new JSONArray();
arr.put(10);
arr.put(20);
arr.put(35);
connectedUser.put("points", arr);
print("connectedUser.length():" + connectedUser.length());
```

btw when connectedUser is empty=={}, connectedUser.toString().length() is 2 because it calculates the string's length